flocker/control/_model.py

```diff
@@ -573,6 +575,7 @@ def __invariant__(self):
                 return (False, '%r is not correct key for %r' % (key, value))
         return (True, "")
 
+    uuid = field(type=UUID, factory=UUID, mandatory=True)
```

Comment:
I kinda hate `factory`.  Code instantiating this type should just pass a `UUID` instance.  Accidentally passing a byte string of the right length shouldn't pass silently.